homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,264 +76,266 @@
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust humidity",
+        unit_of_measurement=PERCENTAGE,
+        sensor_id=SENSOR_HUMIDITY_EXHAUST,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_SUPPLY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Supply airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_EXHAUST,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_BYPASS_STATE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Bypass state",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:camera-iris",
+        sensor_id=SENSOR_BYPASS_STATE,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_DAYS_TO_REPLACE_FILTER,
+        name="Days to replace filter",
+        unit_of_measurement=TIME_DAYS,
+        icon="mdi:calendar",
+        sensor_id=SENSOR_DAYS_TO_REPLACE_FILTER,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_POWER_CURRENT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Power usage",
+        unit_of_measurement=POWER_WATT,
+        sensor_id=SENSOR_POWER_CURRENT,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_POWER_TOTAL,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Energy total",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        sensor_id=SENSOR_POWER_TOTAL,
+        last_reset=dt.utc_from_timestamp(0),
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_PREHEATER_POWER_CURRENT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Preheater power usage",
+        unit_of_measurement=POWER_WATT,
+        sensor_id=SENSOR_PREHEATER_POWER_CURRENT,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_PREHEATER_POWER_TOTAL,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Preheater energy total",
+        unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        sensor_id=SENSOR_PREHEATER_POWER_TOTAL,
+        last_reset=dt.utc_from_timestamp(0),
+    ),
+)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Optional(CONF_RESOURCES, default=[]): vol.All(
-            cv.ensure_list, [vol.In(SENSOR_TYPES)]
+            cv.ensure_list, [vol.In([desc.key for desc in SENSOR_TYPES])]
```

Comment:
Side note: Long term we should remove this option and just allow users to enable/disable entities as wanted via the entity registry, and we set the less important entities as disabled by default. That means the description key will change to be the sensor_id.

Reply:
I want to take a look at converting this to a config flow later, and then this will go away anyway.